source/postprocess/gravity_point_values.cc

```diff
@@ -414,6 +432,47 @@ namespace aspect
             }
         }
 
+      // write quantities in the statistic file
+      const std::string name0("Stokes velocity polynomial degree");
```

Comment:
I would prefer this not to be written into the statistics file, I will expand on this in the main reply.